packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -357,7 +234,7 @@ class PersonalizedRankingPage extends StatelessWidget {
             sliver: SliverList(
               delegate: SliverChildBuilderDelegate(
                 (BuildContext context, int index) =>
-                    personalizedRakingModel.dismissed.first.product == null
+                    personalizedRakingModel.products.first.product == null
```

Comment:
all the personalizedRakingModel strings should be replaced with personalizedRankingModel